.github/scripts/template-zip-autogen.sh

```diff
@@ -12,6 +14,9 @@ TEMPLATE_LIST=(
     bot-msgext.default
 )
 
+# Copy bot code to msgext-bot, except readme and images
+rsync -az --recursive --exclude "*.md" --exclude "*images/*" ./templates/bot/ ./templates/bot-msgext/
```

Comment:
I may need more context about this.  
1. mustache templates stay in bot-msgext folder.
2. copy bot-msgext to bot folder.

how about the msgext folder?

Nit: In addition, what's the reason behind the exclude logic?

Reply:
discussed offline